apps/total-typescript/src/hooks/use-mux-player.tsx

```diff
@@ -33,11 +33,19 @@ type VideoProviderProps = {
   lesson: VideoResource
   path?: string
   muxPlayerRef: any
+  onEnded?: () => Promise<any>
```

Comment:
wanted to be able to track in the local context, specifically to write to database